### PR TITLE
[WIP] fix(feat): add CatBoost predictor. Fixes #1491

### LINKED ISSUE #1491: Catboost serving
Good day. Can I serve a catboost model with kfserving? If yes. Can you point me to an example or explain the process. Thank you

### PR DESCRIPTION
**Pull request by NikeNano**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by NikeNano** [comment]
Create a WIP pr to start to trigger some of the test, still some things I need to fix. Should I split the work in to separate PR:s to make it easier to review?

File: python/catboostserver/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+To run static type checks:
+
+```bash
+mypy --ignore-missing-imports sklearnserver
+```
+An empty result will indicate success.
+
+## Building your own Scikit-Learn Server Docker Image
```
Comment (NikeNano) [inline python/catboostserver/README.md:61]: will update this.

**Comment by NikeNano** [comment]
FYI: will pick this up again now.

**Comment by yuzisun** [comment]
@NikeNano Thanks for the PR, we are redesigning the model serving runtime which allows plugging more model frameworks or custom runtime, please see https://docs.google.com/document/d/1LcnBobEaHTe_WpwqwrJG9N1zIFg61zqV1I9JErp_Eqo for more details.

**Pull request closed without merging**